curriculum/challenges/english/03-front-end-libraries/react-and-redux/use-provider-to-connect-redux-to-react.english.md

```diff
@@ -95,10 +95,10 @@ class DisplayMessages extends React.Component {
   }
   submitMessage() {
     const currentMessage = this.state.input;
```

Comment:
Hii, @RandellDawson  can you please make it more clear.do I have to add currentMessage in state and change it inside setState

Reply:
> Hii, @RandellDawson can you please make it more clear.do I have to add currentMessage in state and change it inside setState

Yes, you have to do that, because trying to reference this.state does not guarantee the correct value would be used.